src/routes/destroy.jsx

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+import { redirect } from "react-router-dom";
+import { deleteContact } from "../components/Contact";
+
+export async function action({ params }) {
+  await deleteContact(params.contactId);
+  return redirect("/");
+}
```

Comment:
- [ ] I do not think this file is related to this particular project, kindly remove it to make your project more professional.